ImperatorToCK3/Data_Files/configurables/replaceable_file_blocks.txt

```diff
@@ -110,6 +110,2060 @@
+											percent >= 0.5
+											title_province = { geographical_region = custom_ep3_restore_rome_eastern_empire }
+										}
+									}
+								}
+								
+							}
+
+							change_liege = {
+								liege = scope:new_holder
+								change = scope:change
+							}
+						}
+					}
+				}
+			}
+		} # End of after
+	}
+
+	# Change Roman Empire title and de jure hierarchy to the WRE title to represent full Empire split
+	replace = {
+		before = {
+		resolve_title_and_vassal_change = scope:change
+		hidden_effect = { set_primary_title_to = title:e_byzantium }
+		} # end of before
+
+		after = {
+		hidden_effect = { set_primary_title_to = title:e_byzantium }
+
+		# IRToCK3: Swap e_roman_empire title with WRE title
+		title:e_roman_empire = {
+			every_in_de_jure_hierarchy = {
+				limit = {
+					tier = tier_kingdom
+				}
+				set_de_jure_liege_title = title:e_western_roman_empire
+			}
+		}
+
+		title:e_western_roman_empire = {
+			change_title_holder = {
+				holder = title:e_roman_empire.holder
+				change = scope:change
+			}
+		}
+
+		root = {
+			set_primary_title_to = title:e_western_roman_empire
+		}
+
+		resolve_title_and_vassal_change = scope:change
+
+		root = {
+			destroy_title = title:e_roman_empire
+		}
+		} # end of after
+	}
+
+}
+
+
+"common/character_interactions/00_character_interactions.txt" = {
+	
+	# offer_vassalization_interaction ai_accept block
+	replace = {
+		before = {
+					OR = {
+						this = culture:greek
+						any_parent_culture_or_above = {
+							this = culture:greek
+						}
+						this = culture:han
+						any_parent_culture_or_above = {
+							this = culture:han
+						}
+					}
+		}# end of before
+
+		after = {
+					OR = { # IRToCK3: Modified to include roman since this is historical admin realm cultures
+						this = culture:greek
+						this = culture:roman
+						any_parent_culture_or_above = {
+							this = culture:greek
+						}
+						any_parent_culture_or_above = {
+							this = culture:roman
+						}
+						this = culture:han
+						any_parent_culture_or_above = {
+							this = culture:han
+						}
+					}
+		} # end of after
+	}
+
+}
+
+
+"common/culture/traditions/07_ep3_traditions.txt" = {
+	
+	# tradition_ep3_imperial_tagmata is_shown block
+	replace = {
+		before = {
+		OR = {
+			this = culture:greek
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			has_cultural_pillar = heritage_byzantine
+		}
+		# DLC check.
+		has_ep3_dlc_trigger = yes
+		NOT = {
+			has_cultural_tradition = tradition_roman_legacy
+		}
+		}# end of before
+
+		after = {
+		OR = { # IRToCK3: Added roman as possible culture
+			this = culture:greek
+			this = culture:roman
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			any_parent_culture_or_above = {
+				this = culture:roman
+			}
+			#has_cultural_pillar = heritage_byzantine # IRToCK3: Removed since this heritage know represents general Hellenistic cultures, so it doesn't really fit
```

Comment:
```suggestion
			#has_cultural_pillar = heritage_byzantine # IRToCK3: Removed since this heritage now represents general Hellenistic cultures, so it doesn't really fit
```